tests-functional/tests/test_firstrun.py

```diff
@@ -0,0 +1,68 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.firstrun import Firstrun
+
+
+@pytest.mark.nondestructive
+def test_open_firstrun_page(base_url, selenium):
+    page = Firstrun(selenium, base_url).open()
+    # ensure all-aboard dialog heading text match
+    assert 'Have you used Firefox in the last 30 days?' == page.question_one_heading  # noqa E501
```

Comment:
Just a warning that checking content can be troublesome, especially if it changes frequently or if you want to test functionality in alternative locales.